docs/ref/contrib/gis/install/geolibs.txt

```diff
@@ -88,9 +88,9 @@ is required.
 
 .. note::
 
-    OS X users are required to install `Apple Developer Tools`_ in order
+    macOS users are required to install `Apple Developer Tools`_ in order
     to compile software from source.  This is typically included on your
-    OS X installation DVDs.
+    macOS installation DVDs and available on its web site.
 
 .. _Apple Developer Tools: https://developer.apple.com/technologies/tools/
```

Comment:
This URL is redirecting to https://developer.apple.com/xcode/ -- should "Apple Developer Tools" be updated to "Xcode"? I think the "This is typically..." sentence could be removed since I think most Macs don't include an optical drive anymore.